G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookFolderStructure.md

```diff
@@ -0,0 +1,28 @@
+# outlookfolder
+
+This structure stores information about the Outlook folder, which was retrieved with the `outlook.getfolder` command. It contains the following fields:
+
+| Field        | Type                                                        | Description                                     |
+| ------------ | ----------------------------------------------------------- | ----------------------------------------------- |
+| `name`       | [text](../../../g1ant.language/structures/textstructure.md) | The name of the folder                          |
+| `folderpath` | [text](../../../g1ant.language/structures/textstructure.md) | The path to the folder                          |
+| `folders`    | [list](../../../g1ant.language/structures/liststructure.md) | The list of subfolders                          |
+| `mails`      | [list](../../../g1ant.language/structures/liststructure.md) | The list of email messages stored in the folder |
+| `unreaded`   | [list](../../../g1ant.language/structures/liststructure.md) | The list of unread messages                     |
```

Comment:
would be great to have information here values of what structure those lists store, for example: ⟦list:mail⟧

Reply:
Because in that case it was misleading (I gave an example with credential variable) but in G1ANT fro example if you want to force a variable to be of list structure containing only integers, you have to do something like this:
`♥listOfMyInts = ⟦list:integer⟧1❚2❚3❚4❚5❚6`
I don't know why it works this way in G1ANT but this is how you have to do it in order to force some structure in lists

Reply:
I thought I've already showed you in some review how to force structures in lists but maybe I'm wrong :)

Reply:
but here you already get a list of elements of mail structure and you don't have to cast it in any way